### PR TITLE
adding epirus chain explorer

### PR DESCRIPTION
**Pull request by jvlkl**

### CODE REVIEW

File: quorum/templates/08-epirus-chain-explorer.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      name:  epirus-chain-explorer-deployment
+      labels:
+        app: epirus
+        tier: backend
+        name: epirus-chain-explorer-deployment
+    spec:
+      securityContext:
+      containers:
+      - name: epirus-api
+        image: web3labs/epirus-free-api:latest
```
Comment (jpbu) [inline quorum/templates/08-epirus-chain-explorer.yaml:25]: I would rather not use latest here which cause problems if there is an update with breaking changes, Image tag/version should be a Helm value (applies to all images used in this file)

File: quorum/values.yaml [hunk 2]
```diff
@@ -7,6 +7,11 @@ geth:
   raftPort: 50401
   verbosity: 3
   gethParams: --permissioned \ --nodiscover \ --nat=none \ --unlock 0 \ --emitcheckpoints \ --rpccorsdomain '*' \ --rpcvhosts '*' \
+epirus:
+  enabled: true
+  port: 3000
```
Comment (jpbu) [inline quorum/values.yaml:12]: Does this port really need to be configurable? Other ports like the one for the API are hard-coded.

File: quorum/templates/08-epirus-chain-explorer.yaml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        ports:
+          - containerPort: 8090
+        env:
+          - name: NODE_ENDPOINT
+            value: http://{{$scope.Values.epirus.node}}.quorum-network.svc.cluster.local:8545
```
Comment (jpbu) [inline quorum/templates/08-epirus-chain-explorer.yaml:30]: This includes the namespace in a hard-coded way, I think if we assume that Epirus is running in the same namespace we don't need the `.<namespace>.svc.cluster.local` suffix

File: quorum/templates/08-epirus-chain-explorer.yaml [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+          - name: MONGO_CLIENT_URI
+            value: mongodb://localhost:27017
+          - name: REINDEX_ENDPOINT
```
Comment (jpbu) [inline quorum/templates/08-epirus-chain-explorer.yaml:33]: Is this really used and does it work? I do not see where the `ingestion` host is defined

File: quorum/templates/08-epirus-chain-explorer.yaml (hunk 1 above)
Comment (jvlkl) [reply quorum/templates/08-epirus-chain-explorer.yaml:25]: Sadly for epirus-web, api and ingestion only the 'latest' tag is available. Changed the image tag for MongoDB to 4.4.5.

File: quorum/values.yaml (hunk 2 above)
Comment (jvlkl) [reply quorum/values.yaml:12]: Removed the port for now, if the need arises we can make everything adjustable.

File: quorum/templates/08-epirus-chain-explorer.yaml (hunk 3 above)
Comment (jvlkl) [reply quorum/templates/08-epirus-chain-explorer.yaml:30]: removed the suffix, works well

File: quorum/templates/08-epirus-chain-explorer.yaml (hunk 4 above)
Comment (jvlkl) [reply quorum/templates/08-epirus-chain-explorer.yaml:33]: removed this env variable as it is not used

**Review by jpbu — APPROVED** [review]

**Pull request merged**